src/clr-angular/forms/radio/radio.module.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+import {CommonModule} from "@angular/common";
+import {NgModule} from "@angular/core";
+
+import {ClrHostWrappingModule} from "../../utils/host-wrapping/host-wrapping.module";
+import {ClrCommonFormsModule} from "../common/common.module";
+
+import {ClrRadio} from "./radio";
+import {ClrRadioContainer} from "./radio-container";
+
+/**
+ * Internal module, please do not export!
```

Comment:
I forget if this compiles correct (TS seems to have an open PR that might indicate this wont work at the moment) but can we put `@private` on this comment for IDEs?

Reply:
I'm not sure what the point of `@private` would be. All that matters here is that `clr-angular/index.ts` doesn't export this, right? If people start importing stuff from source files rather than our index, then they already know they're doing something not supported.